src/components/options/OptionsContainer.js

```diff
@@ -29,6 +28,58 @@ class OptionsContainer extends Component {
     };
   }
 
+  constructor(props) {
+    super(props);
+
+    this.handleChangeDelay = this.handleChangeDelay.bind(this);
+    this.handleClickMinus = this.handleClickMinus.bind(this);
+    this.handleClickPlus = this.handleClickPlus.bind(this);
+    this.confirmChangeDelay = this.confirmChangeDelay.bind(this);
+
+    this.confirmDelayTimeout = null;
+
+    this.state = {
+      delay: props.options.get('timeToClose') ? props.options.get('timeToClose') : 30,
```

Comment:
This feature is enabled from the beginning (sleeping feature) https://github.com/secretin/secretin-lib/blob/master/src/User.js#L327